### ISSUE TITLE
panw related.user contains duplicate user names when client/source and server/destination user name are identical

### ISSUE THREAD

**Issue by willemdh** [elastic/beats#20517]
The `panw.panos` `related.user` field contains duplicate user names when client/source and `server/destination` user name are identical.

The `filebeat-7.8.1-panw-panos-pipeline` should contain logic to prevent adding user names which are already in the `related.user` array

```
  {
    "append": {
      "if": "ctx?.client?.user?.name != null",
      "field": "related.user",
      "value": "{{client.user.name}}"
    }
  },
  {
    "append": {
      "field": "related.user",
      "value": "{{source.user.name}}",
      "if": "ctx?.source?.user?.name != null"
    }
  },
  {
    "append": {
      "field": "related.user",
      "value": "{{server.user.name}}",
      "if": "ctx?.server?.user?.name != null"
    }
  },
  {
    "append": {
      "field": "related.user",
      "value": "{{destination.user.name}}",
      "if": "ctx?.destination?.user?.name != null"
    }
  }
```

**Comment by legoguy1000** [comment]
This has been resolved in 7.12.0. https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/panw/panos/ingest/pipeline.yml

**Comment by legoguy1000** [comment]
I recommend closing this issue.
